docs/route/error-boundary.md

```diff
@@ -4,28 +4,6 @@ title: ErrorBoundary
 
 # `ErrorBoundary`
 
-<docs-warning>The separation of `CatchBoundary` and `ErrorBoundary` has been deprecated and Remix v2 will use a singular `ErrorBoundary` for all thrown Responses and Errors. It is recommended that you opt-into the new behavior in Remix v1 via the `future.v2_errorBoundary` flag in your `remix.config.js` file. Please refer to the [ErrorBoundary (v2)][error-boundary-v2] docs for more information.</docs-warning>
+An `ErrorBoundary` is a React component that renders whenever anything is thrown anywhere on the route, either during rendering or during data loading. This can be a thrown `Response`/`Error` from a loader/action or an `Error` thrown during rendering. This `ErrorBoundary` is an implementation of the React Router []`errorElement`/`ErrorBoundary`][rr-error-boundary].
```

Comment:
I know we talk about this in the RR docs, but I feel like we should least have a few usage examples here as well. In general I don't love that so many of our docs just brush over concepts and link off to RR.

Reply:
good call - added back some docs and also added a missing `isRouteErrorElement` page